src/RiotAPILibraries.ts

```diff
@@ -157,7 +157,24 @@ export default class RiotAPILibraries {
     }
 
     private async getList(message: Discord.Message) {
-        const reply = this.settings.riotApiLibraries.languageList.replace("{languages}", "`" + this.languageList.join(", ") + "`");
+        let applicableLangs = Array.from(this.languageMap.keys());
+
+        if(message.channel.type == 'GUILD_TEXT') { // if this is the server text channel
```

Comment:
error TS2367: This condition will always return 'false' since the types '"text" | "dm" | "news"' and '"GUILD_TEXT"' have no overlap.